docs/index.rst

```diff
@@ -88,12 +94,12 @@ Decorators
 Operators
 ^^^^^^^^^
 - **SetupRayCluster**: Sets up or Updates a ray cluster on kubernetes using a kubeconfig input provided through the Ray connection
-- **DeleteRayCluster**: Deletes and existing Ray cluster on kubernetes using the same Ray connection
+- **DeleteRayCluster**: Deletes an existing Ray cluster on kubernetes using the same Ray connection
 - **SubmitRayJob**: Submits jobs to a Ray cluster using a specified host name and monitors its execution
 
 Triggers
 ^^^^^^^^
-- **RayJobTrigger**: Monitors asynchronous job execution submitted via ``SubmitRayJob`` or using the ``@task.ray()`` decorator and prints real-time logs to the the Airflow UI
+- **RayJobTrigger**: Monitors asynchronous job execution submitted via the ``SubmitRayJob`` operator or using the ``@ray.task()`` decorator and prints real-time logs to the the Airflow UI
```

Comment:
I don't think user need to know about it. shall we delete Triggers section

Reply:
We could remove hook also and just keep operators

Reply:
This section explains the components that are part of this provider..

Hooks and Triggers don't have much value to the user since they work on the backend but I included them for completeness.